p2pserver/message/types/version.go

```diff
@@ -88,5 +90,10 @@ func (this *Version) Deserialization(source *comm.ZeroCopySource) error {
 		return comm.ErrIrregularData
 	}
 
+	this.P.TransportType, eof = source.NextByte()
```

Comment:
the `NextByte` may belong to another message. you need increment `Version`field to handle this